solanches/rest.py

```diff
@@ -131,6 +131,17 @@ def adiciona_destaques(comercio_nome):
     return jsonify(msg), 201
 
 
+@app.route("/comercio/<comercio_nome>/produto/<produto_id>", methods=['DELETE'])
+def remove_produto(comercio_nome, produto_id):
+    try:
+        controller.remove_produto(comercio_nome, produto_id)
```

Comment:
Retorna o json do comércio pra ser possível visualizar a alteração feita
```suggestion
        comercio = controller.remove_produto(comercio_nome, produto_id)
```